CHANGELOG.md

```diff
@@ -1,24 +0,0 @@
-# Changelog
```

Comment:
I guess we can keep the changelog with incrementing version?

Reply:
Oh, ok!